.github/workflows/pkgdown.yaml

```diff
@@ -73,12 +73,17 @@ jobs:
           pkgdown::build_site()
         shell: Rscript {0}
 
+      - name: Create index file
+        if: contains(env.isPush, 'false')
+        run: |
+          cp fake-index.html ./docs/index.html
```

Comment:
It might make more sense to inline `fake-index` in here, using `cat` to or similar to turn it into a file. That would keep everything more self-contained.